src/Utils.js

```diff
@@ -12,6 +12,8 @@ import {
   REACT15,
 } from './version';
 
+export const ITERATOR_SYMBOL = typeof Symbol === 'function' && Symbol.iterator;
```

Comment:
Seems like a great module to put out there!

As far as the tests go, I did consciously chose to test the actual public API that this provides, rather then testing the internal implementation. Though more robust tests are rarely a bad thing! I can add some more tests now that verify the iterator method is there, and only there when `Symbol.iterator` is defined.

Otherwise, I can make sure to add tests using `hasSymbols` once we can use it 😄